src/www/firewall_schedule.php

```diff
@@ -97,6 +97,12 @@
                 }]
       });
     });
+
+    // Decode time range descriptions previously encoded via escape()
+    $('.range-description').each(function(i, description) {
```

Comment:
I think we shouldn't try to present data differently, it might have side affects on "new" data and there's no way of knowing we should present differently.

Reply:
Hi @AdSchellevis , thank you for reviewing the code. I see what you're saying. So if someone intentionally wanted to save a string such as this,

> 12%3A00%20AM%20to%208%3A00%20AM

it should be left as-is. No problem. I'll remove those lines of code and re-commit.  I'll also remove the same code that leverages `decodeURIComponent()` in `firwall_schedule_edit.php` as well.

Thank you.